<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14245" started-at="2019-05-10T08:42:28Z" finished-at="2019-05-10T08:42:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14245" started-at="2019-05-10T08:42:28Z" finished-at="2019-05-10T08:42:43Z">
      <class name="TestingAllure">
        <test-method status="PASS" signature="setup()[pri:0, instance:TestingAllure@1e67a849]" name="setup" is-config="true" duration-ms="3303" started-at="2019-05-10T04:42:28Z" finished-at="2019-05-10T04:42:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="test1()[pri:0, instance:TestingAllure@1e67a849]" name="test1" duration-ms="2698" started-at="2019-05-10T04:42:32Z" finished-at="2019-05-10T04:42:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="test2()[pri:0, instance:TestingAllure@1e67a849]" name="test2" duration-ms="1590" started-at="2019-05-10T04:42:35Z" finished-at="2019-05-10T04:42:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="SKIP" signature="test3()[pri:0, instance:TestingAllure@1e67a849]" name="test3" duration-ms="5420" started-at="2019-05-10T04:42:36Z" finished-at="2019-05-10T04:42:42Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Testing skip.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Testing skip.
	at TestingAllure.test3(TestingAllure.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="PASS" signature="fnl()[pri:0, instance:TestingAllure@1e67a849]" name="fnl" is-config="true" duration-ms="937" started-at="2019-05-10T04:42:42Z" finished-at="2019-05-10T04:42:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fnl -->
      </class> <!-- TestingAllure -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
